2025-03-19 16:48:53,147 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:74]
2025-03-19 16:48:53,265 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1057]
2025-03-19 16:48:59,338 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:74]
2025-03-19 16:48:59,546 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1057]
2025-03-19 16:48:59,919 INFO: Request method: GET [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:672]
2025-03-19 16:49:33,593 INFO: Request method: POST [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:672]
2025-03-19 16:49:33,595 INFO: POST request received for /register [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:675]
2025-03-19 16:49:33,596 INFO: Received Form Data: [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:678]
2025-03-19 16:49:33,598 INFO:   Name: reuben [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:679]
2025-03-19 16:49:33,599 INFO:   Email: mwaurareuben199@gmail.com0797596539 [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:680]
2025-03-19 16:49:33,600 INFO:   Phone: 0797596539 [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:681]
2025-03-19 16:49:33,602 INFO:   Password: @Mwaurareuben01 [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:682]
2025-03-19 16:49:33,603 INFO:   Confirm Password: @Mwaurareuben01 [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:683]
2025-03-19 16:49:33,831 ERROR: Form validation failed during registration. [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:692]
2025-03-19 16:49:33,832 ERROR: Validation errors: {'email': ['Invalid email address.']} [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:693]
2025-03-19 16:49:33,910 INFO: Request method: GET [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:672]
2025-03-19 16:51:32,204 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:74]
2025-03-19 16:51:32,376 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1057]
2025-03-19 16:59:36,341 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 16:59:36,516 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1064]
2025-03-19 16:59:41,395 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 16:59:41,746 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1064]
2025-03-19 16:59:42,387 INFO: Request method: GET [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:673]
2025-03-19 17:00:14,039 WARNING: Session expired or user_id not found in session. [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:749]
2025-03-19 17:00:14,070 INFO: Request method: GET [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:673]
2025-03-19 17:03:38,921 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 17:03:39,119 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1003]
2025-03-19 17:03:44,109 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 17:03:44,302 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1003]
2025-03-19 17:07:47,551 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 17:07:47,727 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1003]
2025-03-19 17:07:53,409 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:75]
2025-03-19 17:07:53,615 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:1003]
2025-03-19 17:12:41,504 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:12:41,782 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:985]
2025-03-19 17:12:46,798 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:12:46,989 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:985]
2025-03-19 17:14:07,764 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:14:07,985 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:985]
2025-03-19 17:14:10,964 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:512]
2025-03-19 17:14:47,416 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:512]
2025-03-19 17:17:50,619 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:17:50,815 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:985]
2025-03-19 17:17:56,385 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:17:56,564 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:985]
2025-03-19 17:18:55,828 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:512]
2025-03-19 17:20:35,140 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:512]
2025-03-19 17:34:28,565 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:34:28,739 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:988]
2025-03-19 17:34:29,132 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:34:39,393 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:34:39,577 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:988]
2025-03-19 17:34:44,943 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:34:45,056 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:988]
2025-03-19 17:34:45,329 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:34:49,698 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:34:52,151 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:35:16,549 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:37:45,836 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:37:46,048 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:988]
2025-03-19 17:37:50,909 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:37:51,126 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:988]
2025-03-19 17:37:51,574 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:37:56,366 INFO: Client connected [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:515]
2025-03-19 17:40:05,922 INFO: MongoDB connection successful [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:71]
2025-03-19 17:40:06,150 INFO: Starting application... [in C:\Users\USER\Documents\reuben\Privacy_Assessment_Tool\assessment_project\my_app.py:996]
